### ISSUE TITLE
Enable logging from user-defined functions 

### ISSUE THREAD

**Issue by devin-petersohn** [modin-project/modin#3122]
It would be useful to allow logging from the user-defined functions called in lower-level operators.

Originally brought up here: https://discuss.modin.org/t/logging-within-a-worker-function/211

**Comment by arunjose696** [comment]
The post in [https://discuss.modin.org](https://discuss.modin.org/t/logging-within-a-worker-function/211) is currently unavailable so answering this assuming the issue is regarding how developers could log from UDFs and ray remote functions.

In ray it is possible to log from UDF by just declaring the logger config inside the remote function, Developers can log from UDFs and ray remote functions called in lower level functions as in the below example.

```
@ray.remote
def remote_function(log_level=logging.INFO):
    logging.basicConfig(filename='modin_udf.log', level=logging.INFO)
    logging.info("This log message will be written to modin_udf.log ")
    return 
```

However if  developer attempts to use modin logging  in the remote function/UDF as below, multiple log directories(.modin/logs/job_**) would be created for each worker executing the remote function.

```
def Udf(df, partition_shape, *dfs):
    from modin.config import LogMode, LogMemoryInterval, LogFileSize
    LogMode.enable()
    ...
    return
```

 So  recommended approach would be to use a different logger(as in the first snippet in this comment) to log from  UDF if user intends to log information from the user defined functions that execute on ray workers.

**Comment by anmyachev** [comment]
@YarShev should we update docs with this information?

**Comment by YarShev** [comment]
Yes, that probably makes sense. @arunjose696, could you please put a paragraph related to this on https://modin.readthedocs.io/en/stable/usage_guide/advanced_usage/modin_logging.html page?
